tools/osv-linter/internal/pkgchecker/ecosystems.go

```diff
@@ -2,26 +2,34 @@ package pkgchecker
 
 import (
 	"fmt"
-	"io"
-	"net/http"
-	"regexp"
-	"slices"
-	"strings"
-
-	"golang.org/x/mod/module"
-	"golang.org/x/mod/semver"
-
-	"github.com/ossf/osv-schema/linter/internal/faulttolerant"
-
-	pep440 "github.com/aquasecurity/go-pep440-version"
-
-	"github.com/tidwall/gjson"
 )
 
 // Ecosystem support is a work in progress.
 var SupportedEcosystems = []string{
 	"Go",
 	"PyPI",
+	"crates.io",
+	"npm",
+	"NuGet",
+	"RubyGems",
+	"Packagist",
+	"Pub",
+	"Hackage",
+	"Maven",
+}
+
+// EcosystemBaseURLs maps ecosystems to their base API URLs.
+var EcosystemBaseURLs = map[string]string{
```

Comment:
There's an opportunity to DRY this out a bit, and we can use the keys of `EcosystemBaseURLs` to infer what is now duplicated into `SupportedEcosystems`.